fe/src/main/java/org/apache/doris/common/util/TimeUtils.java

```diff
@@ -268,4 +269,11 @@ public static String checkTimeZoneValidAndStandardize(String value) throws DdlEx
         }
         throw new DdlException("Parse time zone " + value + " error");
     }
+
+    // return the number of days of give year.month
+    public static int getNumberDaysOfMonth(int year, int month) {
+        Calendar calendar = Calendar.getInstance();
+        calendar.set(year, month + 1, 0);
```

Comment:
why plus one? `getNumberDaysOfMonth(2020, 1)` will return 29, and the function name is confused

Reply:
This function is useless, i will remove it